activerecord/test/cases/finder_test.rb

```diff
@@ -202,10 +202,15 @@ def test_exists_with_nil_arg
     assert_equal true, Topic.first.replies.exists?
   end
 
-  # Ensure +exists?+ runs without an error by excluding distinct value.
+  # ensures +exists?+ runs without an error by excluding distinct value
   # See https://github.com/rails/rails/pull/26981.
-  def test_exists_with_order_and_distinct
-    assert_equal true, Topic.order(:id).distinct.exists?
+  def test_exists_with_distinct
+    assert_equal true, Topic.distinct.exists?
```

Comment:
This test case expects that `exists?` runs without `ERROR:  for SELECT DISTINCT, ORDER BY expressions must appear in select list` even if both `order` and `distinct` are specified. It should not be changed.
See https://github.com/rails/rails/pull/26981.